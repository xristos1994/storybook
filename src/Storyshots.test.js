import initStoryshots from '@storybook/addon-storyshots';
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';
const path = require('path');

const storybookUrl = process.env.STORYBOOK_SOURCE_ENV === 'static'
  ? `file://${path.resolve(__dirname, '../storybook-build')}`
  : process.env.STORYBOOK_SOURCE_ENV === 'prod'
    ? 'https://ck-storybook.netlify.app'
    : `http://localhost:${process.env.STORYBOOK_PORT_ENV}`


initStoryshots({suite: 'Snapshots storyshots'});
initStoryshots({suite: 'Image storyshots', test: imageSnapshot({storybookUrl})});