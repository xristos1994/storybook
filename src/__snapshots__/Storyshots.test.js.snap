// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots storyshots Button Button With Controls 1`] = `
<button
  className="button primary"
  disabled={false}
  onClick={[Function]}
>
  Test Button
</button>
`;

exports[`Snapshots storyshots Button Primary Button 1`] = `
<button
  className="button primary fullWidth"
  disabled={false}
  onClick={[Function]}
>
  Primary
</button>
`;

exports[`Snapshots storyshots Button Secondary Button 1`] = `
<button
  className="button secondary fullWidth"
  disabled={false}
  onClick={[Function]}
>
  Secondary
</button>
`;

exports[`Snapshots storyshots Counter Counter With Controls 1`] = `
<div
  className="counter"
>
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    -
  </button>
  <h3>
    6
  </h3>
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    +
  </button>
  <br />
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;

exports[`Snapshots storyshots Counter Default Counter 1`] = `
<div
  className="counter"
>
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    -
  </button>
  <h3>
    0
  </h3>
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    +
  </button>
  <br />
  <button
    className="button primary"
    disabled={false}
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;

exports[`Snapshots storyshots MainPage Default Main Page 1`] = `
<div
  className="mainPage"
>
  <div
    className="counterContainer"
  >
    <button
      className="button secondary"
      disabled={false}
      onClick={[Function]}
    >
      Disable Counter
    </button>
    <div
      className="counter"
    >
      <button
        className="button primary"
        disabled={false}
        onClick={[Function]}
      >
        -
      </button>
      <h3>
        10
      </h3>
      <button
        className="button primary"
        disabled={false}
        onClick={[Function]}
      >
        +
      </button>
      <br />
      <button
        className="button primary"
        disabled={false}
        onClick={[Function]}
      >
        Reset
      </button>
    </div>
  </div>
  <div
    className="timerContainer"
  >
    <div
      className="timer"
    >
      <h3>
        00h 00m 00s 
      </h3>
      <div
        className="actions"
      >
        <button
          className="button primary"
          disabled={false}
          onClick={[Function]}
        >
          Start
        </button>
        <button
          className="button primary"
          disabled={false}
          onClick={[Function]}
        >
          Stop
        </button>
        <button
          className="button primary"
          disabled={false}
          onClick={[Function]}
        >
          Reset
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshots storyshots Timer Default Timer 1`] = `
<div
  className="timer"
>
  <h3>
    00h 00m 00s 
  </h3>
  <div
    className="actions"
  >
    <button
      className="button primary"
      disabled={false}
      onClick={[Function]}
    >
      Start
    </button>
    <button
      className="button primary"
      disabled={false}
      onClick={[Function]}
    >
      Stop
    </button>
    <button
      className="button primary"
      disabled={false}
      onClick={[Function]}
    >
      Reset
    </button>
  </div>
</div>
`;
